android/app/src/main/java/app/organicmaps/util/SharingUtils.java

```diff
@@ -234,6 +234,9 @@ public static void shareFile(Context context, ActivityResultLauncher<SharingInte
       // Properly set permissions for intent, see
       // https://developer.android.com/reference/androidx/core/content/FileProvider#include-the-permission-in-an-intent
       intent.setFlags(Intent.FLAG_GRANT_READ_URI_PERMISSION);
+
+      intent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
```

Comment:
Should it be inside `if (context instanceof Application)`, as it is done in other apps?

Reply:
In this case context is Activity.

Reply:
Does it allow to come back into Organic Maps after the file is exported?

Reply:
Yes. After exporting we return to Organic Maps (checked it for exporting to Google Drive and local deevice)

Reply:
There are several more places in the code where this function is called. Did you test it on different Android versions too? Does it work on Android 5?

Reply:
i tested it in send bug report functionality.
It works correcctly on Android 10
On Android 5 send e-mail and savind to documents work too. Saving to Google Drive i can't check

Reply:
Should FLAG_ACTIVITY_NEW_TASK be used when a single bookmark or POI is shared from the Place Page?

Reply:
On current version works correctly